clock/app/views/employees/_form.html.erb

```diff
@@ -0,0 +1,104 @@
+
+      <ul>
+        <% employee.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  
+  </div>  </div>
+  <% end %>
+
+  <div class="container">
+  <div class="row">
+    <div class="col-sm-2">
+      <div class="row">
+        <div class="col-sm-12">
+          <img src="https://picsum.photos/200/200" class="img-thumbnail" />
+        </div>
+      </div>
+    </div>
+    <div class="col-sm-10">
+<%= form_with(model: employee, local: true, class: "form-horizontal") do |form| %>
+  
+  <div class="form-group row">
+    <%= form.label :name, :class => "col-sm-2 col-form-label" %>
+    <div class="col-sm-10">
+    <%= form.text_field :name, placeholder: "First name, Last name" %>
+    </div>
+  </div>
+
+  <div class="form-group row">
+    <%= form.label :email, :class => "col-sm-2 col-form-label" %>
+    <div class="col-sm-10">
+    <%= form.text_field :email, placeholder: "abc@mail.com" %>
+    </div>
+  </div>
+
+  <div class="form-group row">
+    <%= form.label :position, :class => "col-sm-2 col-form-label" %>
+    <div class="col-sm-10">
+    <%= form.text_field :position, placeholder: "CEO" %>
+    </div>
+  </div>
+
+  <div class="form-group row">
+    <%= form.label :phone, :class => "col-sm-2 col-form-label" %>
+    <div class="col-sm-10">
+    <%= form.text_field :phone, placeholder: "+52 (312) 000 0000" %>
+    </div>
+  </div>
+
+  <div class="form-group row">
+    <%= form.label :address, :class => "col-sm-2 col-form-label" %>
+    <div class="col-sm-10">
+    <%= form.text_field :address, placeholder: "Address Name # 101, Colima" %>
+    </div>
+  </div>
+
+  <div class="form-group row">
+    <%= form.label :nip, :class => "col-sm-2 col-form-label" %>
+    <div class="col-sm-10">
+    <%= form.text_field :nip, placeholder: "1234" %>
+    </div>
+  </div>
+
+  <div class="form-group row">
+    <%= form.label :work_days, :class => "col-sm-2 col-form-label" %>
+    <div class="col-sm-10 row">
+    <% ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'].each do |work_day| -%>
+      <%= check_box_tag "employee[work_days][#{work_day}]", work_day, @employee.work_days&.any?(work_day), :class => "col-sm-2 col-form-label", :checked => true  %>
+      
+      <%= label_tag "employee[work_days][#{work_day}]", work_day, :class => "col-sm-2 col-form-label"  %>
+
+    <% end -%>    
+    </div>
+  </div>
+
+  <div class="form-group row">
+    <%= form.label "Work  hours", :class => "col-sm-2 col-form-label" %>
+     <div class="col-sm-10">
+    <%= form.time_select :enter_time %> &nbsp; to &nbsp; <%= form.time_select :departure_time %>
+    </div>
+  </div>
+
+
+
+  <div class="form-group row">
+  <div class="col-sm-10 offset-sm-2">
+    <%= form.submit class: 'btn btn-primary float-right' %>
+  </div>
+  </div>
+<% end %>
+</div>
+</div>
+
+</div>
```

Comment:
pls add and extra line at the end of this file
and delete the blank line above this `div` tag
and check the indentation there because there are two divs closing at the same level